community/kernel/src/main/java/org/neo4j/unsafe/batchinsert/DirectRecordAccess.java

```diff
@@ -236,7 +234,7 @@ public void commit()
         }
 
         List<DirectRecordProxy> directRecordProxies = new ArrayList<>( batch.values() );
-        Collections.sort(directRecordProxies, ( o1, o2 ) -> -o1.getKey().compareTo( o2.getKey() ) );
+        Collections.sort(directRecordProxies, ( o1, o2 ) -> Long.compare( -o1.getKey(), o2.getKey() ) );
```

Comment:
No need to go through `Collections`. Can just call sort on list

`directRecordProxies.sort( ( o1, o2 ) -> Long.compare( -o1.getKey(), o2.getKey() ) );`